pyproject.toml

```diff
@@ -41,15 +41,14 @@ Changelog = "https://pypdf.readthedocs.io/en/latest/meta/CHANGELOG.html"
 
 [project.optional-dependencies]
 full = [
-    "cryptography; python_version >= '3.7'",
-    "PyCryptodome; python_version == '3.6'",
-    "Pillow>=8.0.0",
+    "cryptography; python_version >= '3.8'",
+    "Pillow>=10.3.0",
 ]
 crypto = [
-    "cryptography; python_version >= '3.7'",
-    "PyCryptodome; python_version == '3.6'",
+    "cryptography; python_version >= '3.8'",
+    "PyCryptodome; python_version >= '3.6'",
```

Comment:
This is wrong and should be discussed in another PR.

Reply:
Ok to discuss it later

Reply:
I was aware that both would be installed. I wanted to still show compatbility with both libraries although cryptography is recommanded. I revert and let you propose a PR for this

Reply:
I had started on a PR to remove `PyCryptodome` altogether (including the file that references it), which I can look to finish up in next week or so.